src/keylessBackup/index.test.ts

```diff
@@ -96,7 +96,7 @@ describe(getEncryptedMnemonic, () => {
 
   it('throws if non 200 response', async () => {
```

Comment:
nit: change test name here to be a bit more descriptive now that we discriminate between different types of non-200s?